configs/api-platform.json

```diff
@@ -1,7 +1,15 @@
 {
   "index_name": "api-platform",
   "start_urls": [
-    "https://api-platform.com/docs"
+    {
+      "url": "https://api-platform.com/docs"
```

Comment:
This URL should be placed at the end since the first one matching will be applied, none version will fit otherwise

Reply:
Done ;)